src/Type/Definition/ArgumentsTrait.php

```diff
@@ -73,4 +78,22 @@ protected function buildArguments(string $typeName, array $rawArguments): array
 
         return $arguments;
     }
+
+    /**
+     * @param string $name
+     * @return ArgumentInterface|null
+     */
+    public function getArgument(string $name): ?ArgumentInterface
+    {
+        return $this->arguments[$name] ?? null;
+    }
+
+    /**
+     * @param string $name
+     * @return bool
+     */
+    public function hasArgument(string $name): bool
+    {
+        return isset($this->arguments[$name]);
```

Comment:
Maybe array_key_exists would be better?

Reply:
Yas and No. `isset` is the language opcode and x2-x3 faster than `array_key_exists`. So if guarantees are required that `null` is also a valid argument value, then it should be replaced by:
```
return isset($this->arguments[$name]) || \array_key_exists($name, $this->arguments);
```

Reply:
Yeah I was mostly thinking about the "null is valid" case

Reply:
fixed https://github.com/digiaonline/graphql-php/pull/351/commits/eb13a44b3c9ce08176b6dd061f7b9c7a9193cf6a